obal/__init__.py

```diff
@@ -125,7 +125,7 @@ def main(cliargs=None):
     # this needs to be global, as otherwise PlaybookCLI fails
     # to set the verbosity correctly
     from ansible.utils.display import Display
-    global display
+    global display  # pylint: disable=C0103,W0601
```

Comment:
what were these warnings?

Reply:
[C0103](http://pylint-messages.wikidot.com/messages:c0103) is an invalid name. Globals are supposed to be in all caps but this is because ansible requires it. I now see that there are more pylint failures and a better way to clear it. Going to revisit this. Will also look at adding the checks to CI.